### PR TITLE
Translate Queueing a Series of State Updates page

### LINKED ISSUE #449: Queueing a Series of State Updates
Page: https://id.react.dev/learn/queueing-a-series-of-state-updates

Assignee: @milhamm

### PR DESCRIPTION
**Pull request by mrevanzak**
Closes #449 

## Description

Translate the Queueing a Series of State Updates page.
Page URL: [Queueing a Series of State Updates](https://id.react.dev/learn/queueing-a-series-of-state-updates)

### CODE REVIEW

File: src/content/learn/queueing-a-series-of-state-updates.md [hunk 1]
```diff
@@ -47,29 +47,33 @@ h1 { display: inline-block; margin: 10px; width: 30px; text-align: center; }
...
+**React tidak melakukan pengelompokkan pada beberapa *event* yang disengaja, seperti klik**--setiap klik ditangani secara terpisah. Pastikan bahwa React hanya melakukan pengelompokan ketika aman untuk dilakukan. Ini memastikan bahwa, misalnya, jika klik tombol pertama menonaktifkan *form*, klik kedua tidak akan mengirimkannya lagi.
 
-## Updating the same state multiple times before the next render {/*updating-the-same-state-multiple-times-before-the-next-render*/}
+## Memperbarui state yang sama beberapa kali sebelum render selanjutnya {/*updating-the-same-state-multiple-times-before-the-next-render*/}
 
 It is an uncommon use case, but if you would like to update the same state variable multiple times before the next render, instead of passing the *next state value* like `setNumber(number + 1)`, you can pass a *function* that calculates the next state based on the previous one in the queue, like `setNumber(n => n + 1)`. It is a way to tell React to "do something with the state value" instead of just replacing it.
```
Comment (rendiputra) [inline src/content/learn/queueing-a-series-of-state-updates.md:72]:
```suggestion
```

Forgot to delete the translated paragraph.

**Review by rendiputra — CHANGES_REQUESTED** [review]
Forgot to delete the translated paragraph.

cc @mrevanzak

**Comment by mrevanzak** [comment]
hi @rendiputra  ,I already change the word according to your review. Please review it again.

File: src/content/learn/queueing-a-series-of-state-updates.md [hunk 2]
```diff
@@ -47,29 +47,32 @@ h1 { display: inline-block; margin: 10px; width: 30px; text-align: center; }
 
 </Sandpack>
 
-However, as you might recall from the previous section, [each render's state values are fixed](/learn/state-as-a-snapshot#rendering-takes-a-snapshot-in-time), so the value of `number` inside the first render's event handler is always `0`, no matter how many times you call `setNumber(1)`:
+Namun, seperti yang mungkin Anda ingat dari bagian sebelumnya, [nilai *state* setiap *render* adalah tetap](/learn/state-as-a-snapshot#rendering-takes-a-snapshot-in-time), sehingga nilai `number` di dalam *event handler* *render* pertama selalu `0`, tidak peduli berapa kali Anda memanggil `setNumber(1)`:
+
+```js
```
Comment (rendiputra) [inline src/content/learn/queueing-a-series-of-state-updates.md:52]: Double syntax highlighting

```suggestion
```

**Review by rendiputra — CHANGES_REQUESTED** [review]
Double syntax highlighting

**Comment by mrevanzak** [comment]
hi @rendiputra ,I already change the word according to your review. Please review it again.

File: src/content/learn/queueing-a-series-of-state-updates.md [hunk 3]
```diff
@@ -47,29 +47,31 @@ h1 { display: inline-block; margin: 10px; width: 30px; text-align: center; }
 
 </Sandpack>
 
-However, as you might recall from the previous section, [each render's state values are fixed](/learn/state-as-a-snapshot#rendering-takes-a-snapshot-in-time), so the value of `number` inside the first render's event handler is always `0`, no matter how many times you call `setNumber(1)`:
+Namun, seperti yang mungkin Anda ingat dari bagian sebelumnya, [nilai *state* setiap *render* adalah tetap](/learn/state-as-a-snapshot#rendering-takes-a-snapshot-in-time), sehingga nilai `number` di dalam *event handler* *render* pertama selalu `0`, tidak peduli berapa kali Anda memanggil `setNumber(1)`:
+
 
 ```js
```
Comment (resir014) [inline src/content/learn/queueing-a-series-of-state-updates.md:53]:
```suggestion
Namun, seperti yang mungkin Anda ingat dari bagian sebelumnya, [nilai *state* setiap *render* adalah tetap](/learn/state-as-a-snapshot#rendering-takes-a-snapshot-in-time), sehingga nilai `number` di dalam *event handler* *render* pertama selalu `0`, tidak peduli berapa kali Anda memanggil `setNumber(1)`:

```js
```

Additional line here should be removed. This will introduce an unnecessary diff which will disrupt our translation sync workflow.

**Review by resir014 — CHANGES_REQUESTED** [review]
@mrevanzak Only one small change from me at the moment.

**Comment by resir014** [comment]
Peer review needed, cc: @mhaidarhanif @zainfathoni

**Comment by mrevanzak** [comment]
hi @resir014 ,I already change the word according to your review. Please review it again. Thank youuu

File: src/content/learn/queueing-a-series-of-state-updates.md [hunk 4]
```diff
@@ -47,29 +47,30 @@ h1 { display: inline-block; margin: 10px; width: 30px; text-align: center; }
...
+## Memperbarui state yang sama beberapa kali sebelum render selanjutnya {/*updating-the-same-state-multiple-times-before-the-next-render*/}
 
-It is an uncommon use case, but if you would like to update the same state variable multiple times before the next render, instead of passing the *next state value* like `setNumber(number + 1)`, you can pass a *function* that calculates the next state based on the previous one in the queue, like `setNumber(n => n + 1)`. It is a way to tell React to "do something with the state value" instead of just replacing it.
 
-Try incrementing the counter now:
+Ini bukanlah penggunaan yang umum, tetapi jika Anda ingin memperbarui variabel *state* yang sama berulang kali sebelum *render* selanjutnya, alih-alih mengoper nilai *state* selanjutnya seperti `setNumber(number + 1)`, Anda dapat mengoper *function* yang menghitung *state* selanjutnya berdasarkan nilai sebelumnya pada antrean, seperti `setNumber(n => n + 1)`. Ini adalah cara untuk memberi tahu React untuk "melakukan sesuatu dengan nilai *state*" daripada hanya menggantinya.
```
Comment (resir014) [inline src/content/learn/queueing-a-series-of-state-updates.md:71]: Second empty line also exists here. Please remove it.

**Review by resir014 — CHANGES_REQUESTED** [review]
Only a small review while we wait for a peer review from @mhaidarhanif + @zainfathoni.

File: src/content/learn/queueing-a-series-of-state-updates.md [hunk 5]
```diff
@@ -225,60 +225,60 @@ h1 { display: inline-block; margin: 10px; width: 30px; text-align: center; }
...
+#### Perbaiki hitungan permintaan {/*fix-a-request-counter*/}
 
-You're working on an art marketplace app that lets the user submit multiple orders for an art item at the same time. Each time the user presses the "Buy" button, the "Pending" counter should increase by one. After three seconds, the "Pending" counter should decrease, and the "Completed" counter should increase.
+Anda bekerja pada aplikasi pasar seni yang memungkinkan pengguna mengirimkan beberapa pesanan untuk item seni pada saat yang sama. Setiap kali pengguna menekan tombol "Buy", hitungan "Pending" harus bertambah satu. Setelah tiga detik, hitungan "Pending" harus berkurang, dan hitungan "Completed" harus bertambah.
 
-However, the "Pending" counter does not behave as intended. When you press "Buy", it decreases to `-1` (which should not be possible!). And if you click fast twice, both counters seem to behave unpredictably.
+Akan tetapi, hitungan "Pending" tidak berperilaku seperti yang diharapkan. Ketika Anda menekan "Buy", ia berkurang menjadi `-1` (yang seharusnya tidak mungkin!). Dan jika Anda mengklik cepat dua kali, kedua hitungan tampaknya berperilaku diluar kendali.
```
Comment (mhaidarhanif) [inline src/content/learn/queueing-a-series-of-state-updates.md:279]:
```suggestion
Akan tetapi, penghitung "Pending" tidak berperilaku seperti yang diharapkan. Ketika Anda menekan "Buy", ia berkurang menjadi `-1` (yang seharusnya tidak mungkin!). Dan jika Anda mengklik cepat dua kali, kedua hitungan tampaknya berperilaku diluar kendali.
```

File: src/content/learn/queueing-a-series-of-state-updates.md [hunk 6]
```diff
@@ -47,29 +47,29 @@ h1 { display: inline-block; margin: 10px; width: 30px; text-align: center; }
...
 
-**React does not batch across *multiple* intentional events like clicks**--each click is handled separately. Rest assured that React only does batching when it's generally safe to do. This ensures that, for example, if the first button click disables a form, the second click would not submit it again.
+**React tidak melakukan pengelompokkan pada beberapa *event* yang disengaja, seperti klik**--setiap klik ditangani secara terpisah. Pastikan bahwa React hanya melakukan pengelompokan ketika aman untuk dilakukan. Ini memastikan bahwa, misalnya, jika klik tombol pertama menonaktifkan *form*, klik kedua tidak akan mengirimkannya lagi.
 
-## Updating the same state multiple times before the next render {/*updating-the-same-state-multiple-times-before-the-next-render*/}
+## Memperbarui state yang sama beberapa kali sebelum render selanjutnya {/*updating-the-same-state-multiple-times-before-the-next-render*/}
```
[the remark this answers is not in the document]
Comment (mrevanzak) [reply src/content/learn/queueing-a-series-of-state-updates.md:68]: @mhaidarhanif heading bukannya tidak perlu seperti itu?

[the remark this answers is not in the document]
Comment (mhaidarhanif) [reply src/content/learn/queueing-a-series-of-state-updates.md:68]: @mrevanzak Ohiya lupa. Tidak perlu italic. Ini abaikan saja.

**Review by mhaidarhanif — APPROVED** [review]

**Review by resir014 — APPROVED** [review]
